modyn/models/tokenizers/distill_bert_tokenizer.py

```diff
@@ -0,0 +1,16 @@
+import torch
+from transformers import DistilBertTokenizer
+
+
+class DistilBertTokenizerTransform:
+    def __init__(self, max_token_length: int = 300) -> None:
+        self.max_token_length = max_token_length
+        self.tokenizer = DistilBertTokenizer.from_pretrained("distilbert-base-uncased")
+
+    def __call__(self, sample: str) -> torch.Tensor:
+        tokens = self.tokenizer(
+            sample, padding="max_length", truncation=True, max_length=self.max_token_length, return_tensors="pt"
+        )
+        data = torch.stack((tokens["input_ids"], tokens["attention_mask"]), dim=2)
+        data = torch.squeeze(data, dim=0)  # First shape dim is always 1
```

Comment:
is it? should we assert that and add a short reason why we expect it to be that way?

Reply:
This is also code that is not mine that I tend to trust

Reply:
Well we should not blindly merge code. In any case, do these models have some form of test? I think for the existing DLRM and ResNes Models, we at least check we can initialize the models. For small models here, we might want to also assert the output is as expected  (shape-wise)

Reply:
I added some comments to the class

Reply:
Also some tests